app/javascript/components/widget-controller.jsx

```diff
@@ -2,10 +2,12 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import gql from 'graphql-tag';
 import { Query } from 'react-apollo';
+import * as R from 'ramda';
```

Comment:
Nit: you can just import `pathOr` by destructuring instead of all functions:
`import { pathOr } from 'ramda';`